sigal/gallery.py

```diff
@@ -440,6 +443,13 @@ def thumbnail(self):
         return None
 
     @property
+    def rand_thumbnail(self):
+        try :
+            return join(self.name, random.choice(self.medias).thumbnail)
+        except IndexError:
+            return self.thumbnail
```

Comment:
Yep but as you use a `join` it can give a bad url (mostly for Windows, on which I'm not sure if everything works correctly but ...).